bundles/org.openhab.binding.pjlinkdevice/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -103,6 +104,12 @@
 		<description>Select the video mute status</description>
 	</channel-type>
 
+	<channel-type id="audioAndVideoMute">
```

Comment:
In addition to provide upgrade instructions for the thing, please also regenerate the i18n file to include your new channel: `mvn i18n:generate-default-translations`

Reply:
As these are small changes, i added both updated translation and instructions. Please proceed with the review if possible.